web/styles/popovers.css

```diff
@@ -287,7 +287,7 @@ ul {
    to user info popovers ONLY when they are opened from the right
    sidebar; otherwise, it will have the user-card-popover class
    instead. */
-.user_popover {
+.user_sidebar {
```

Comment:
This class name is too generic -- this is a popover, not the sidebar itself. `user-sidebar-popover` would match current patterns.

Reply:
Fixed!